RELEASE-NOTES.txt

```diff
@@ -11,6 +11,7 @@
 * [**] Block Editor: Media uploads that failed due to lack of internet connectivity automatically retry once a connection is re-established [https://github.com/wordpress-mobile/WordPress-iOS/pull/22238]
 * [**] Block Editor: Manually retrying a single failed media upload will retry all failed media uploads in a post [https://github.com/wordpress-mobile/WordPress-iOS/pull/22240]
 * [*] [internal] Drop iOS 14 Support: Remove deprecated UIDocumentPickerViewController API [#22286]
+* [**] Site creation is re-enabled for WordPress users with no sites. [#22415]
```

Comment:
I wonder if this should be marked internal?

Reply:
I am really not sure. If you think that makes more sense I'll do that.